e2e_correctness/query_neo_mem.py

```diff
@@ -297,3 +301,88 @@ def neo4j_get_graph(neo4j_driver: neo4j.BoltDriver) -> Graph:
     json_data = get_neo4j_data_json(neo4j_driver)
     logger.debug("Building the graph from Neo4j JSON data")
     return create_graph_neo4j_json(json_data)
+
+
+# additions for path testing
+def sort_dict(dict):
+    keys = list(dict.keys())
+    keys.sort()
+    sorted_dict = {i: dict[i] for i in keys}
+    if "id" in sorted_dict:
+        sorted_dict.pop("id")
```

Comment:
This part is not part of sorted_dict, I think you should move that to other function

Reply:
So, this function should just sort dict, and you return the sorted dict from this function and then remove id